sigs.yaml

```diff
@@ -717,6 +717,15 @@ sigs:
     owners:
     - https://raw.githubusercontent.com/kubernetes-sigs/alibaba-cloud-csi-driver/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes/cloud-provider-alibaba-cloud/master/OWNERS
+    meetings:
+    - description: Regular Alibaba Cloud Subproject Meeting
+      day: Tuesday
+      time: "4:00"
+      tz: PT (Pacific Time)
```

Comment:
4 am makes me suspect that pacific time is not the right time zone for the intended audience. I would suggest either using UTC or the time zone of the intended audience.

Reply:
OK, we will use the time zone of the intended audience.

Reply:
thetimezoneconverter seems not to recognize CST(China Standard Time), so I use UTC instead.

Reply:
you can use [`China`](http://www.thetimezoneconverter.com/?t=20:00&tz=China) instead of `CST`.

Reply:
@idealhack 
Hello, it showed：
![image](https://user-images.githubusercontent.com/5154334/71232479-1a4d5e80-232d-11ea-8b91-a963e65b4f28.png)

Reply:
Hotel or Hong Kong would probably work if they are acceptable. UTC is obviously fine.